### PR TITLE
Filtering and infinite scroll

### PR DESCRIPTION
**Pull request by danthecodingman**
Filtering query setup and infinite scrolling on the home page

### CODE REVIEW

File: fbuapp/Views/PostTableView.m [hunk 1]
```diff
@@ -68,7 +68,16 @@ -(void)customInit
     
     self.refreshControl = [[UIRefreshControl alloc] init];
     [self.refreshControl addTarget:self action:@selector(fetchPosts) forControlEvents:UIControlEventValueChanged];
-    [self addSubview:self.refreshControl];
+     [self addSubview:self.refreshControl];
```
Comment (eazheng) [inline fbuapp/Views/PostTableView.m]:
```suggestion
    [self addSubview:self.refreshControl];
```

File: fbuapp/AppDelegate.m [hunk 2]
```diff
@@ -39,31 +39,43 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
...
-    else {
-        //If user if not logged in, take them to LogViewController
-        LogViewController *logViewController = [[LogViewController alloc] init];
-        logViewController.modalPresentationStyle = UIModalPresentationFullScreen;
-        logViewController.modalTransitionStyle = UIModalTransitionStyleCoverVertical;
-
-        UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:logViewController];
-    }
-    self.window.rootViewController = navigationController;
+//    if ([PFUser currentUser] != nil) {
```
Comment (belchercd) [inline fbuapp/AppDelegate.m]: There is an updated version for this with tab control inside if statement. Tab bar now has control of which page user goes to when logged in.

File: fbuapp/Views/PostTableView.h [hunk 3]
```diff
@@ -23,9 +24,11 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithUserId:(NSString *)userId;
 
 @property (nonatomic, weak) id <PostTableViewDelegate> delegate;
-@property (strong, nonatomic) NSArray * posts;
-
-
+@property (strong, nonatomic) NSMutableArray * posts;
```
Comment (belchercd) [inline fbuapp/Views/PostTableView.h]: your spacing for pointers is .... interesting

File: fbuapp/Views/PostTableView.m [hunk 4]
```diff
@@ -68,7 +68,16 @@ -(void)customInit
...
+    
+    CGRect frame = CGRectMake(0, self.contentSize.height, self.bounds.size.width, InfiniteScrollActivityView.defaultHeight);
+    self.loadingMoreView = [[InfiniteScrollActivityView alloc] initWithFrame:frame];
+    self.loadingMoreView.hidden = true;
+    [self addSubview:self.loadingMoreView];
+
+    UIEdgeInsets insets = self.contentInset;
```
Comment (belchercd) [inline fbuapp/Views/PostTableView.m:78]: does this not need a pointer?

**Review by danthecodingman — COMMENTED** [review]
Looks good

**Pull request merged**